package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@blinkmobile/server-cli",
-  "description": "develop, test and deploy server-side HTTPS endpoints with BlinkMobile",
+  "description": "CLI to develop, test and deploy server-side HTTPS endpoints with BlinkMobile",
```

Comment:
Try to keep package.json:description the same as that description line near the top of README.md